pkg/imagedistributor/mount.go

```diff
@@ -108,13 +118,17 @@ func (c ImagerMounter) Mount(imageName string) ([]ClusterImageMountInfo, error)
 	return imageMountInfos, nil
 }
 
-func (c ImagerMounter) Umount(imageMountInfo []ClusterImageMountInfo) error {
+func (c ImagerMounter) Umount(imageName string, imageMountInfo []ClusterImageMountInfo) error {
 	for _, info := range imageMountInfo {
 		err := c.Mounter.Umount(info.MountDir)
 		if err != nil {
 			return fmt.Errorf("failed to umount %s:%v", info.MountDir, err)
 		}
 	}
+	// delete all mounted images
+	if err := fs.FS.RemoveAll(filepath.Join(mountBaseDir, formatImageName(imageName))); err != nil {
```

Comment:
how about extracting a public method to return the mount root directory for the specified image name?